src/CommonDBTM.php

```diff
@@ -6340,28 +6340,36 @@ public static function displayFullPageForItem(
             $menus = $menus[Session::getCurrentInterface()];
         }
 
-        if (static::isNewID($id)) {
-            // New item, check create rights
-            if (!static::canCreate()) {
-                static::displayAccessDeniedPage($menus);
-                return;
-            }
+        if (static::$record_type === static::RECORD_ITEMIZED) {
```

Comment:
Instead of introducting a new property, maybe it could be simplier to accept a `null` value in `$id` parameter.

Reply:
A null ID is valid for the `isNewID` check. `displayFullPageForItem` would still need extra logic to handle null values before the int cast and new ID check.

Having a flag property in the class seems less likely to cause confusion and could be used in other areas if special handling needs done where usage may be dynamic. In this case, we know we are talking about Configs because of the specific front file, but there may be other uses where the itemtype could be different at runtime.